firestore/tests/unit/v1/test_watch.py

```diff
@@ -735,6 +735,38 @@ class DummyDoc(object):
             doc_tree, doc_map, delete_changes, add_changes, update_changes
         )
         self.assertEqual(updated_map, doc_map)  # no change
+        
+    def test__compute_snapshot_deletes_w_real_comparator(self):
+        from google.cloud.firestore_v1.watch import WatchDocTree
+
+        doc_tree = WatchDocTree()
+
+        class DummyDoc(object):
+            update_time = mock.sentinel
+
+        def _comparator(doc1, doc2):
+            if doc1.field > doc2.field:
+                return 1
+            if doc1.field < doc2.field:
+                return -1
+            return 0
```

Comment:
Coverage fails for this function (because it isn't being called).  Let's just delete it and pass through an opaque object below for the comparator, e.g.:

```python
inst = self._makeOne(comparator=object())
```